Dockerfile.catalyst

```diff
@@ -31,7 +31,7 @@ ENV LD_LIBRARY_PATH="/src/compiled/lib" C_INCLUDE_PATH="/src/compiled/include"
 RUN	git clone -b catalyst https://github.com/DDVTECH/mistserver.git \
 	&& mkdir -p mistserver/build \
 	&& cd mistserver/build \
-	&& git show 9e7e080e3a5dbef8fa4164fe7351cf6178911389 \
+	&& git show a773d1270 \
```

Comment:
is this to invalidate caching or you really want to see the diff?

Reply:
cache invalidation, yeah. I guess I could have done `checkout` but I kind of wanted it to be "this version or later please"